src/Illuminate/Routing/ResourceRegistrar.php

```diff
@@ -228,7 +228,7 @@ protected function getGroupResourceName($prefix, $resource, $method)
      */
     public function getResourceWildcard($value)
     {
-        return str_replace('-', '_', $value);
+        return camel_case($value);
```

Comment:
Now it format value to snake case like, so we must naming our variables like **$blog_article**, **$task_choice** (for resources like 'blog-aricle', 'task-choice') etc. 
For resources like 

```php
$router->resource('taskChoice', 'TaskChoiceController');
```

all will be fine, but i can't create resource 

```php
$router->resource('task-choice', 'TaskChoiceController');

```

and save pretty code style (for variables naming)
And i propose to change it to camel case naming.